pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -402,9 +423,23 @@ func NewLegacyRegistry() *LegacyRegistry {
 			}
 		})
 	registry.registerPriorityConfigProducer(EvenPodsSpreadPriority,
-		func(args ConfigProducerArgs, plugins *config.Plugins, _ *[]config.PluginConfig) {
+		func(args ConfigProducerArgs, plugins *config.Plugins, pluginConfig *[]config.PluginConfig) {
 			plugins.PreScore = appendToPluginSet(plugins.PreScore, podtopologyspread.Name, nil)
 			plugins.Score = appendToPluginSet(plugins.Score, podtopologyspread.Name, &args.Weight)
+			if feature.DefaultFeatureGate.Enabled(features.DefaultPodTopologySpread) {
+				// Configure plugin to use List default spreading, unless SelectorSpreadPriority was added.
+				for _, e := range *pluginConfig {
+					if e.Name == podtopologyspread.Name {
+						return
+					}
+				}
+				*pluginConfig = append(*pluginConfig, config.PluginConfig{
+					Name: podtopologyspread.Name,
+					Args: &config.PodTopologySpreadArgs{
+						DefaultingType: config.ListDefaulting,
+					},
+				})
```

Comment:
What does it mean to have `ListDefaulting` without setting `DefaultConstraints`?

Reply:
It means no default constraints.

Reply:
ditto, lets add a comment clarifying the semantics in this case.